### PR TITLE
fix broken source URLs in MariaDB easyconfigs

### PR DESCRIPTION
**Pull request by vanzod**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/m/MariaDB/MariaDB-10.3.10-foss-2018b.eb [hunk 1]
```diff
@@ -38,6 +35,9 @@ checksums = [
     '8295837e623f6c782e1d64b00e0877ea98cce4bf8846755bb86c8a7732797c19',  # MariaDB-10.1.13-link-rt-for-jemalloc.patch
 ]
 
+# MariaDB pulls in submodules which don't have releases.
+# We download the submodule revisions from their repos.
+# determine commit of additional sources via https://github.com/MariaDB/server/tree/10.3
```
Comment (boegel) [inline easybuild/easyconfigs/m/MariaDB/MariaDB-10.3.10-foss-2018b.eb]: @vanzod Please move this back up?

File: easybuild/easyconfigs/m/MariaDB/MariaDB-10.0.21-GNU-4.9.3-2.25.eb [hunk 2]
```diff
@@ -8,8 +8,9 @@ description = """MariaDB An enhanced, drop-in replacement for MySQL."""
 
 toolchain = {'name': 'GNU', 'version': '4.9.3-2.25'}
 
+source_urls = ['https://downloads.mariadb.org/interstitial/mariadb-%(version)s/source']
 sources = [SOURCELOWER_TAR_GZ]
-source_urls = ['https://downloads.mariadb.org/f/mariadb-10.0.21/source/']
+checksums = ['cad1471777c86f2be3050ed71360397c3999481cb8c00c317da077399eecf7ad']
```
Comment (boegel) [inline easybuild/easyconfigs/m/MariaDB/MariaDB-10.0.21-GNU-4.9.3-2.25.eb]: This checksum is different from what is was before (perhaps also for the others). Any idea what happened?

File: easybuild/easyconfigs/m/MariaDB/MariaDB-10.3.10-foss-2018b.eb (hunk 1 above)
Comment (vanzod) [reply easybuild/easyconfigs/m/MariaDB/MariaDB-10.3.10-foss-2018b.eb]: This was inadvertently modified by `inject-checksum`. Reverting it back as it was.

File: easybuild/easyconfigs/m/MariaDB/MariaDB-10.0.21-GNU-4.9.3-2.25.eb (hunk 2 above)
Comment (vanzod) [reply easybuild/easyconfigs/m/MariaDB/MariaDB-10.0.21-GNU-4.9.3-2.25.eb]: Well, the new URL is actually wrong. Pushing the correct one now.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 10 out of 10 (8 easyconfigs in this PR)
node2028.delcatty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/0b9f0677e2d5e46da2acea562a980420 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @vanzod!

**Pull request merged**